lib/snippets/deploy-to-gke.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
```

Comment:
Where does this script get invoked from? Does it care whether this file is `deploy-to-gke.rb` or `deploy-to-gke`?